### PR TITLE
ON-9306 # added FormTemplate

### PR DESCRIPTION
**Pull request by Zaxist**

### CODE REVIEW

File: flow-typed/forms.js [hunk 1]
```diff
@@ -428,6 +428,14 @@ declare type Form = {
   tags: Array<string>,
 }
 
+declare type FormTemplate = {
+  id: number,
+  name: string,
+  description: string,
```
Comment (simonmarklar) [inline flow-typed/forms.js:434]: description is optional in the builder so it should probs be optional here

File: typescript/forms.d.ts [hunk 2]
```diff
@@ -432,6 +432,14 @@ export interface Form {
   tags: string[]
 }
 
+export interface FormTemplate {
+  id: number,
+  name: string,
+  description: string,
```
Comment (simonmarklar) [inline typescript/forms.d.ts:438]: same

File: flow-typed/forms.js [hunk 3]
```diff
@@ -428,6 +428,14 @@ declare type Form = {
   tags: Array<string>,
 }
 
+declare type FormTemplate = {
+  id: number,
+  name: string,
+  description?: string,
+  elements: FormElement[],
+  tags: string[],
```
Comment (mymattcarroll) [inline flow-typed/forms.js:436]: can we please add, `isInfoPage`, `isMultiPage`, `createdAt`, `updatedAt`

Comment (mymattcarroll) [reply flow-typed/forms.js:436]: And could we please split this into NewFormTemplate and FormTemplate

**Pull request merged**